azerothcore/azerothcore-wotlk
# fix(DB/creature): Correct Newman's Landing NPC positions and vendor items.

## Changes Proposed:
-  Updates positions and vendor items of NPCs from Newman's Landing to more accurately reflect blizzlike values.

## Issues Addressed:

- Closes unreported issue.

## SOURCE:

Wrath Classic PTR sniff.

## Tests Performed:

- Observed in-game.
[These are the values currently proposed in the PR.](https://user-images.githubusercontent.com/38561765/187608732-a3315d22-22b0-4a00-9b50-a4320d3f7b8f.png)
[This is what it looks like on the PTR.](https://user-images.githubusercontent.com/38561765/187608775-05cfb464-76c3-47d5-bb73-57144e854727.png)
[These are the current values.](https://user-images.githubusercontent.com/38561765/187608813-4e22d30f-62cb-4d40-b842-41cefaaf076f.png)

## How to Test the Changes:


1. Run the query.
2. `.go xyz -6378 1268 7`
3. Observe the changes.

## Known Issues and TODO List:


- [x] Currently WPP can't process PTR sniffs entirely. Couple things that are missing are coordinates and equipped item IDs. As noted in the file, those need to be reparsed once able, hence why this is a draft PR.
- [ ] According to what info there is in the sniff, Shelly is supposed to be playing the oneshot_fishing emote, but this was not observed in-game, so it was not added in the PR.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
